tests/test_core.py

```diff
@@ -865,6 +865,16 @@ def test_options_from_code_are_prioritized(self):
         # Then
         assert config_io.options == {"option_1": False, "option_2": True, "option_3": True, "option_4": True}
 
+    @pytest.mark.unit
+    def test_transform_class_names_to_dataset_names(self):
+        # Given
+        camel_case_strings = ["TestStringABC", "TestString", "ThisIsAnotherTest", "AbstractS3Test", "YetAnotherGREATTest"]
+        # When
+        transformed_strings = [DynamicDataIO._transform_class_names_to_dataset_names(s) for s in camel_case_strings]  # pylint: disable=W0212
```

Comment:
I see what you've done here! But parameterized tests would make more sense

Reply:
Done: https://github.com/VorTECHsa/dynamicio/pull/24/commits/30ea96f0e6a088bfb455f01f1eed08e98f8b9f8a